deps/build.jl

```diff
@@ -34,6 +34,10 @@ if is_apple()
     provides(Homebrew.HB, "homebrew/science/hdf5", hdf5, os=:Darwin)
 end
 
+if Sys.KERNEL === :FreeBSD
```

Comment:
We actually don't need any of these branches, I was just matching style in this PR.